### ISSUE TITLE
Preferences button not visible

### ISSUE THREAD

**Issue by nightsky30** [z3ntu/RazerGenie#50]
Maybe it is me and I'm just really missing it, but I don't see a preferences button anywhere.  My layout looks like the following (without the downloaded images):

![prefs_gone](https://user-images.githubusercontent.com/6024315/57498656-5520eb80-72a2-11e9-9f2a-a12784f061d7.png)

In https://github.com/z3ntu/RazerGenie/issues/43#issuecomment-461806253 it is mentioned there is a button in the lower left, but I don't see one.  I'd really like to be able to download the image files for my devices, but cannot access the setting in preferences.

For reference I'm running Xubuntu 19.04 (XFCE), OpenRazer 2.5.0, and RazerGenie 0.8.1

**Comment by nightsky30** [comment]
Update:

The button is actually there, but invisible!!!  I noticed the Daemon Version label was slightly skewed to the right compared to the reference image used above.  I decided to click in that blank area, and a box popped up.

![Screenshot_2019-05-09_21-40-55](https://user-images.githubusercontent.com/6024315/57498869-2fe0ad00-72a3-11e9-9566-bb3516e3fdfc.png)

![Screenshot_2019-05-09_21-40-10](https://user-images.githubusercontent.com/6024315/57498877-3707bb00-72a3-11e9-8016-ed084a4bf710.png)

**Comment by z3ntu** [comment]
I'm using the `configure` icon from the icon theme, which apparently isn't available on your system (I've also seen this problem on Windows where there are no "normal" icon themes)

**Comment by nightsky30** [comment]
At least the area is clickable and functioning.  Is this something that should be fixed upstream say with the icon theme, Qt5, or XFCE distro package?  I'll gladly open an upstream issue :)

**Comment by nightsky30** [comment]
If anyone else finds their way here aggravated and wondering how to set the QT_QPA_PLATFORMTHEME variable to qt5ct or something else:

Xubuntu sets QT_QPA_PLATFORMTHEME in:

`/etc/X11/Xsession.d/56xubuntu-session`

I changed the following to remedy the error I was getting:

```
# QT5 apps to use GTK style
# export QT_QPA_PLATFORMTHEME=gtk2
export QT_QPA_PLATFORMTHEME=qt5ct
```

This did not fix the missing icon issue.  Looking into raising an issue with https://github.com/shimmerproject/elementary-xfce

**Comment by ochosi** [comment]
So the icon name is just "configure"? If it's not shown in elementary-xfce it's probably also not in Gnome's icon theme, from where elementary-xfce inherits all missing icons. That may be a bigger problem than the icon missing in elementary-xfce.

Is that a standard qt icon name?
At least the freedesktop standard doesn't know that name: https://specifications.freedesktop.org/icon-naming-spec/latest/ar01s04.html

Alternatively you could also consider to switch to a standard icon name (especially if it's not a qt standard), e.g. "document-properties"

**Comment by z3ntu** [comment]
Yes, in Breeze it's just `configure`:
![image](https://user-images.githubusercontent.com/3768500/62041685-80114080-b1fc-11e9-8226-d51be03b52f1.png)

How about `preferences-system` ? The icon packs listed at https://commons.wikimedia.org/wiki/Comparison_of_icon_sets have a fitting icon for the preferences.

`preferences-other` would also be an option but it seems that less icon packs support that one.

**Comment by ochosi** [comment]
preferences-system should work out fine!

**Comment by rios0rios0** [comment]
Hello guys.
I have the same behavior.

My system: Ubuntu 18.04.03 LTS (I don't have any Icon Pack, and default appearance settings)
RazerGenie Version: 0.8.1
OpenRazer Daemon Version: 2.7.0

Thanks!
![Screen](https://user-images.githubusercontent.com/32651906/73291004-1fc87f00-41de-11ea-83b4-73a9e4acfadf.png)

**Comment by z3ntu** [comment]
@rios0rios0 The change is not included in RazerGenie 0.8.1. I have some changes that I want to include in the next release (a rewrite more or less) but I haven't got it finished yet, so the new release will still take a while, sorry.
